packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -132,6 +139,13 @@ function CustomViewShell(props: TCustomViewShellProps) {
     notificationsGlobalRef,
     notificationsPageRef,
   });
+  const themeOverrides = useMemo(
+    () => ({
+      ...themesOverrides.default,
+      ...customViewsThemesOverrides.default,
+    }),
+    []
+  );
```

Comment:
Nit: we can also define this as a static variable outside the component, no need to use memo.

Reply:
Yes. Sorry.

Updated here: 1b5f4702b075c058d749fef1287db758ab9d6088